### PR TITLE
fixing location search 

### PR DESCRIPTION
**Pull request by NShani**
[https://issues.openmrs.org/browse/FM-176](url)

### CODE REVIEW

File: api/src/main/java/org/openmrs/module/fhir/api/impl/LocationServiceImpl.java [hunk 1]
```diff
@@ -83,7 +84,7 @@ public Location getLocation(String id) {
 		List<org.openmrs.Location> omrsLocations = Context.getLocationService().getLocations(name);
 		List<Location> locationList = new ArrayList<Location>();
 		for (org.openmrs.Location location : omrsLocations) {
-			if (name.equalsIgnoreCase(location.getName())) {
+			if (StringUtils.startsWithIgnoreCase(location.getName(),name)){
```
Comment (harsha89) [inline api/src/main/java/org/openmrs/module/fhir/api/impl/LocationServiceImpl.java]: need space after ,

Comment (NShani) [inline api/src/main/java/org/openmrs/module/fhir/api/impl/LocationServiceImpl.java]: added a space after ,

**Comment by harsha89** [comment]
@NShani can you add a test case as well?

**Comment by NShani** [comment]
@harsha89  Sure. I will add.

**Comment by dkayiwa** [comment]
@harsha89 @surangak is the description of this module correct? https://github.com/openmrs/openmrs-module-fhir/blob/master/pom.xml#L24

**Comment by harsha89** [comment]
@dkayiwa oopz needed fix that. I will  do it.

**Pull request merged**